panoptes_aggregation/tests/extractor_tests/test_question_extractor.py

```diff
@@ -116,11 +116,11 @@
 }
 
 TestSinglePluck = ExtractorTest(extractors.question_extractor,
-    single_pluck_classification,
-    single_pluck_expected,
-    "Test pluck field functionality with a question extractor",
-    kwargs={'pluck': single_pluck_keys},
-    test_name='TestSinglePluck')
+                                single_pluck_classification,
+                                single_pluck_expected,
+                                "Test pluck field functionality with a question extractor",
+                                kwargs={'pluck': single_pluck_keys},
+                                test_name='TestSinglePluck')
```

Comment:
Don't add the extra spaces here (this is one part of PEP8's style I don't agree with.  I would rather the style be:
```python
TestSinglePluck = ExtractorTest(
    extractors.question_extractor,
    single_pluck_classification,
    ...
)
```

I think it looks cleaner than having a large blank space on each line.

Reply:
Fixed! Ready for merging whenever you have the chance!